### ISSUE TITLE
rfc20: grow support

### ISSUE THREAD

**Issue by dongahn** [flux-framework/rfc#231]
Spins off from #230. There is an idea of adding an additional `R` into JOBID schema to support "grow" in the same way as "shrink"

A key question:

> Then the complementary "grow" directory could contain chunks that are added.
While we are here, maybe we can also has this out a bit as this is what @milroy will soon need.

I don't think adding an additional R is difficult. But what is currently difficult would be how to do this under the original JOBID. In particular, flux job submit will always generate a new JOBID. Do you think there is an easy path to to generate a new R under the same JOBID using flux job submit|flux mini interface?

**Comment by garlick** [comment]
When we discussed this before, I think a new "grow" interface was proposed but maybe that is out of scope for RFC 20?

**Comment by dongahn** [comment]
Yeah something like flux grow or similar can be a solution. I will change this to flux core issue.
